### PR TITLE
Events - Adding Clean Architecture Workshop

### LINKED ISSUE #599: 📄 Events - Clean Architecture Workshop
https://www.ssw.com.au/ssw/Events/Training/Clean-Architecture-Workshop.aspx

As per my conversation with @camillars and @bradystroud , we think all SuperPower pages should be under the route of /events and Internship will remain under /training and remove all the content related to COVID on the page. 
```[tasklist]
### Tasks
- [x] Content migration
- [x] Mega menu update
- [x] Redirection in Front Door
```

### PR DESCRIPTION
**Pull request by amankumarrr**
Fixes #599 

Affected routes:

- `/events/clean-architecture-workshop`

### CODE REVIEW

**Review by Hona — CHANGES_REQUESTED** [review]
old photo of me, please use the latest :)

See the current version: https://www.ssw.com.au/ssw/Events/Training/Clean-Architecture-Workshop.aspx

or google photos

File: content/events/clean-architecture-workshop.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
... demands of users have changed the approach to building enterprise applications. Many challenges exist and just getting started can be a daunting prospect. Let's change that now.\n\nThis hands-on workshop will provide practical guidance and recommendations. We will cover architecture, technologies, tools and frameworks. We will examine strategies for organizing your projects, folders and files. We will design a system that is simple to build and maintain - now and well into the future.\n\nYou will leave this hands-on workshop having developed the foundation and structure for your next enterprise application, along with the knowledge to take your enterprise application development to the next level.\n\n<CustomImage src=\"/images/verticalImageLayout/cloud-architecture-diagram.png\" altText=\"clean architecture diagram\" height={800} width={980} />\n\n## Who Should Attend\n\nAll developers who would like to learn more about Clean Architecture and/or .NET.\n\nExperience: Knows C#\n\n## Computer Setup\n\nYou will need a computer with the latest versions of the following tools installed:\n\n.NET 7 SDK (latest)\n\nWSL2\n\nDocker Desktop\n\nVisual Studio 2022 / JetBrains Rider / VS Code\n"
```
Comment (Harry-Ross) [inline content/events/clean-architecture-workshop.mdx]: Remove the COVID part as per the issue

File: content/events/clean-architecture-workshop.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    presenterList:
+      - presenter: content/presenters/daniel-mackay.mdx
+      - presenter: content/presenters/luke-parker.mdx
+    otherEvent:
+      title: 1-Day Clean Architecture Superpowers Tour.
+      eventURL: >-
+        https://www.ssw.com.au/ssw/Events/Training/Clean-Architecture-Superpowers-Tour.aspx
+    _template: PresenterBlock
+  - prose: true
+    centerAlignedText: true
```
Comment (Harry-Ross) [inline content/events/clean-architecture-workshop.mdx:39]: Should this be centre aligned? I tried it with left align on mine and I thought it looked better - but up to you

**Review by Harry-Ross — CHANGES_REQUESTED** [review]

File: content/events/clean-architecture-workshop.mdx (hunk 1 above)
Comment (amankumarrr) [reply content/events/clean-architecture-workshop.mdx]: Done

File: content/events/clean-architecture-workshop.mdx (hunk 2 above)
Comment (amankumarrr) [reply content/events/clean-architecture-workshop.mdx:39]: the issue is with the title which always remains in the center.

**Review by Hona — COMMENTED** [review]
Is the image intentionally that low quality? I'd suggest using the original, but if its on purpose all good.

I'd rename cloud-architecture-diagram to clean-architecture-diagram

**Review by Hona — APPROVED** [review]
My stuff is resolved - wait till an actual review is done though :)

File: content/events/clean-architecture-workshop.mdx (hunk 2 above)
Comment (Harry-Ross) [reply content/events/clean-architecture-workshop.mdx:39]: Yeah, that looks OK I reckon because its a heading - but up to you

**Review by Harry-Ross — APPROVED** [review]
lgtm 🚢

**Pull request merged**